flow/aggregates.py

```diff
@@ -69,7 +67,6 @@ def aggregator_function(jobs):
         else:
             self._is_aggregate = True
```

Comment:
We can just set this in the `groupsof` classmethod without needing to set an attribute to the function.

Reply:
I'm not sure if I follow this. The `groupsof` method returns an instance of `aggregator`.
But since we're going to introduce the tag attribute, hence I think I'll simplify this logic.